content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -157,116 +191,293 @@ Automatic reordering rules, enabled by setting the reordering rule's :guilabel:`
       :align: center
       :alt: Show automatic reordering rule from the Reordering Rule page.
 
-If the :guilabel:`Buy` route is selected, then an :abbr:`RFQ (Request for Quotation)` is generated.
-To view and manage :abbr:`RFQs (Requests for Quotation)`, navigate to :menuselection:`Purchase app
---> Orders --> Requests for Quotation`.
+If the :guilabel:`Buy` route is selected, then an |RFQ| is generated. To view and manage |RFQs|,
```

Comment:
This and the following paragraphs should be moved above the tip and example blocks. They look too lost in the UI as-is.